apps/designer/app/canvas/features/wrapper-component/wrapper-component.tsx

```diff
@@ -34,16 +33,22 @@ export const WrapperComponentDev = ({
   const [, setSelectedElement] = useSelectedElement();
   const isEditing = editingInstanceId === instance.id;
   const [editableRefCallback, editableProps] = useContentEditable(isEditing);
-  const focusRefCallback = useEnsureFocus();
 
   const refCallback = useCallback(
     (element) => {
-      if (isEditing) editableRefCallback(element);
-      focusRefCallback(element);
+      if (isEditing) {
```

Comment:
this is the wrong place to do it, wrapper-component is decoupled from editable using useContentEditable hook. if the only thing that needs focus is content editable, then focusing should be done inside editableRefCallback